code/modules/projectiles/projectile.dm

```diff
@@ -63,6 +64,18 @@
 
 /obj/item/projectile/proc/Range()
 	range--
+	if(seeking_Angle_change && !QDELETED(original) && can_hit_target(original, permutated, TRUE))
+		Angle = SimplifyDegrees(Angle)
+		var/Angle_temp = Angle - 180
+		var/Angle_to_target = Get_Angle(src, original) - 180
+		var/Angle_difference = Angle_to_target - Angle_temp
+		if(SimplifyDegrees(Angle_temp) == SimplifyDegrees(Angle_to_target + 180) || SimplifyDegrees(Angle_temp + 180) == SimplifyDegrees(Angle_to_target)) //going directly opposite
+			Angle_difference = pick(seeking_Angle_change, -seeking_Angle_change)
+		else if(Angle_difference < -seeking_Angle_change)
+			Angle_difference = -seeking_Angle_change
+		else if(Angle_difference > seeking_Angle_change)
+			Angle_difference = seeking_Angle_change
+		Angle = SimplifyDegrees(Angle + Angle_difference)

```

Comment:
I was wrong, it behaves incorrectly for > 180 derp
this article covers it and has some illustrations that helped me
https://jibransyed.files.wordpress.com/2013/09/yxi24.jpg 
https://jibransyed.wordpress.com/2013/09/05/game-maker-gradually-rotating-an-object-towards-a-target/